test/functional/Java8andUp/src/org/openj9/test/java/lang/ref/Test_PhantomReference.java

```diff
@@ -42,7 +42,7 @@ protected void doneSuite() {
 	@Test
 	public void test_get() {
 		ReferenceQueue rq = new ReferenceQueue();
-		bool = new Boolean(false);
+		bool = Boolean.valueOf(false);
 		PhantomReference pr = new PhantomReference(bool, rq);
```

Comment:
Similar concern about caching of Boolean.valueOf() and changing the test as the lifetime of the Boolean is now much longer

Reply:
There's only one use here, so I don't think it matters.

Reply:
Only 1 use in this test but are you sure no other code in the class library is using `Boolean.valueOf(false)`?  The PhantomRef presumable will indicate when the bool has been collected and that may never happen due to using a cached version,